pelicun/uq.py

```diff
@@ -2019,46 +1999,43 @@ def inverse_transform(self, values: np.ndarray) -> np.ndarray:
         # extrapolate).
         # note: swapping the roles of x_i and y_i for inverse
         # interpolation
-        result = np.interp(values, y_i, x_i)
-
-        return result
+        return np.interp(values, y_i, x_i)
 
 
 class EmpiricalRandomVariable(RandomVariable):
-    """
-    Empirical random variable.
-
-    """
+    """Empirical random variable."""
 
     __slots__: list[str] = ['_raw_samples']
 
     def __init__(
         self,
         name: str,
-        raw_samples: np.ndarray,
+        theta: np.ndarray,
         truncation_limits: np.ndarray | None = None,
         f_map: Callable | None = None,
         anchor: BaseRandomVariable | None = None,
-    ):
+    ) -> None:
+        """Instantiate an Empirical random variable."""
         if truncation_limits is None:
             truncation_limits = np.array((np.nan, np.nan))
         super().__init__(
             name=name,
-            theta=raw_samples,
+            theta=theta,
             truncation_limits=truncation_limits,
             f_map=f_map,
             anchor=anchor,
         )
         self.distribution = 'empirical'
         if not np.all(np.isnan(truncation_limits)):
-            raise NotImplementedError(
-                f'{self.distribution} RVs do not support truncation'
-            )
+            msg = f'{self.distribution} RVs do not support truncation'
+            raise NotImplementedError(msg)
 
-        self._raw_samples = np.atleast_1d(raw_samples)
+        self._raw_samples = np.atleast_1d(theta)
```

Comment:
I believe this should be `_raw_sample` (singular) because it is a single sample.

Reply:
See earlier comment on renaming.

Reply:
I don't understand, why couldn't we rename `self._raw_samples` to `self._raw_sample`? Currently, uq.py only has `raw_samples` variables. Just remove the `s` from the end and it should be fine.
Or, please let me know what I am not seeing.

Reply:
Oops, you are right. Earlier I got confused and thought I had addressed this comment after working on the `uni_samples` issue. I renamed these variables/attributes to `raw_sample` and `_raw_sample`. 12d0f434